scripts/manage-flaky-tests.js

```
@@ -0,0 +1,933 @@
+    switch (command) {
+      case 'detect':
+        await manager.detectFlakyTests(options);
+        break;
+        
+      case 'list':
+        manager.listFlakyTests(options);
+        break;
+        
+      case 'quarantine':
+        const testName = options.test_name || args.find(arg => !arg.startsWith('--'));
+        const testSuite = options.test_suite || args.find((arg, i) => !arg.startsWith('--') && i > 0);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)